fireworks/core/rocket.py

```diff
@@ -42,32 +46,30 @@
 __date__ = "Feb 7, 2013"
 
 
-def do_ping(launchpad, launch_id):
+def do_ping(launchpad: LaunchPad, launch_id: int) -> None:
```

Comment:
Is this `launch_id` `int` or `str`? Went with `int` but wasn't sure.

Reply:
launch_id is an int, thanks!